common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -713,16 +713,19 @@ def login_analytics(strategy, auth_entry, current_partial=None, *args, **kwargs)
 
 
 @partial.partial
-def associate_by_email_if_login_api(auth_entry, backend, details, user, current_partial=None, *args, **kwargs):
+def associate_user_by_email(auth_entry, backend, details, user, current_partial=None, *args, **kwargs):
```

Comment:
Don't we want to apply https://github.com/edx/edx-platform/pull/25935/commits/0219edd6bbb318002a56fce227213e5f864ba1b6 as well?

Reply:
No, that change is [specific to Python 3.8](https://stackoverflow.com/a/57819001/5512292)

Reply:
Hm, since we will upgrade python when we upgrade edx-platform (I guess), this is not an issue then.